From 327176ac36892d068e821a1c496ca0862024f02b Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 23 Apr 2015 10:44:30 +1000
Subject: [PATCH] hwdb: add HP X1000 DPI info

Device name is PixArt, but it's sold as a HP brand.

https://bugs.freedesktop.org/show_bug.cgi?id=90142
(cherry picked from commit e8043cd5fe283ff3023f98c15a2f09328805efab)
---
 hwdb/70-mouse.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb/70-mouse.hwdb b/hwdb/70-mouse.hwdb
index 93ee4d9fa6..6f06e17091 100644
--- a/hwdb/70-mouse.hwdb
+++ b/hwdb/70-mouse.hwdb
@@ -125,6 +125,14 @@ mouse:usb:v0461p4d16:name:USB Optical Mouse:
  MOUSE_DPI=500@125
 
 ##########################################
+# HP
+##########################################
+
+# HP X1000
+mouse:usb:v093ap2510:name:PixArt USB Optical Mouse:
+ MOUSE_DPI=1000@125
+
+##########################################
 # Lenovo
 ##########################################
 
